src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,27 @@
+package seedu.addressbook.data.tag;
+
+import java.util.ArrayList;
+
+import seedu.addressbook.data.person.Person;
+
+public class Tagging {
+
+	private Person person;
```

Comment:
Indentation problem here? Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.